how-to-guides/KyloDatasources.rst

```diff
@@ -0,0 +1,80 @@
+========================
+Kylo Datasources Guide
+========================
+
+Introduction
+====================
+
+Kylo can manage the creation and usage of Nifi RDBMS data source configurations, through a simple `Data Source UI <http://localhost:8400/index.html#!/datasources>`_.
+
+JDBC
+=============
+
+.. note:: Permissions for the jars are separate per the type of process running.
+
+Locations
+-----------
+
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $NIFI_HOME/nifi-<version>/lib    | Needed by Nifi for the DBCPConnectionPool. Can set a custom path, but need to set `Database Driver Location(s)`  |
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $KYLO_HOME/kylo-services/plugin  | Needed by Kylo in the schema discovery (Data Ingestion)                                                          |
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $KYLO_HOME/kylo-services/lib     | Needed by Kylo wrangler (Visual Query / Data Transformation) with Spark SQL                                      |
```

Comment:
So with the local or yarn-client Spark master the jar will need to be added to --driver-class-path. With yarn-cluster then it should be added to --jars. Here's the documentation on the supported properties in spark.properties: http://kylo.readthedocs.io/en/latest/installation/KyloSparkProperties.html